src/containers/Editor.js

```diff
@@ -13,8 +14,10 @@ import { extractStates } from '../transforms/Module';
 
 import { findAvailableKey, createSafeKeyFromName } from '../utils/keys';
 import { StateTemplates, ModuleTemplates } from '../templates/Templates';
+import { BasicTutorial, EditTutorial } from '../templates/Tutorial';
 
 import './Editor.css';
+import './Joyride.scss';
```

Comment:
If we are just using scss as a way to get to Joyride's styles in node_modules, we could do this instead:

```javascript
import '../../node_modules/react-joyride/lib/react-joyride-compiled.css'
```

Bringing in scss and only using it on one line of code seems a little wasteful, unless you think we should start doing stuff in scss.